Kickstarter-iOS/Views/Controllers/ProjectUpdatesViewController.swift

```diff
@@ -44,7 +46,27 @@ internal final class ProjectUpdatesViewController: WebViewController {
 
     self.viewModel.outputs.goToSafariBrowser
       .observeForControllerAction()
-      .observeValues { [weak self] in self?.goToSafariBrowser(url: $0) }
+      .observeValues { [weak self] in
+        self?.goToSafariBrowser(url: $0)
+    }
+
+    self.viewModel.outputs.makePhoneCall
+      .observeForUI()
+      .observeValues { [weak self] number in
+        self?.call(number: number)
+      }
+
+    self.viewModel.outputs.showMailCompose
+      .observeForUI()
+      .observeValues { [weak self] recipient in
+        self?.openMailComposer(recipient: recipient)
+    }
+
+    self.viewModel.outputs.showNoEmailError
+      .observeForUI()
+      .observeValues { [weak self] alertController in
+        self?.present(alertController, animated: true)
+      }
```

Comment:
I think we have some weird spacing here too!

Reply:
🤦‍♂️